thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js

```diff
@@ -354,7 +364,7 @@ export default Ember.Component.extend({
       return {
         timestamps: [entities[urn].start, entities[urn].end || entities[urn].start],
         values: [val, val],
-        color: entities[urn].color,
+        color: getColor(entities[urn].color),
```

Comment:
please also delete the code in timeseries-chart that applies a CSS class from the color string (lines 73-75)